sound/soc/sof/intel/mtl.h

```diff
@@ -62,7 +62,7 @@
 #define MTL_DSP_IRQSTS_IPC		BIT(0)
 #define MTL_DSP_IRQSTS_SDW		BIT(6)
 
-#define MTL_DSP_PURGE_TIMEOUT_US	20000000 /* 20s */
+#define MTL_DSP_PURGE_TIMEOUT_US	500000 /* 0.5s */
```

Comment:
Should we use the existing `HDA_DSP_INIT_TIMEOUT_US` then?

Reply:
yes fixed.